app/forms/idv/api_image_upload_form.rb

```diff
@@ -473,5 +475,14 @@ def store_failed_images(client_response, doc_pii_response)
     def image_resubmission_check?
       IdentityConfig.store.doc_auth_check_failed_image_resubmission_enabled
     end
+
+    def phone_with_camera
+      case params[:phone_with_camera]
+      when 'true'
+        true
+      when 'false'
+        false
+      end
```

Comment:
I missed it in the original PR, but my favorite trick to coerce strings to bools is:
```suggestion
      params[:phone_with_camera].to_s == 'true'
```